node/src/test/scala/coop/rchain/node/configuration/commandline/ConfigMapperSpec.scala

```diff
@@ -212,6 +212,7 @@ class ConfigMapperSpec extends FunSuite with Matchers {
           numberOfActiveValidators = 111111,
           genesisBlockNumber = 222,
           posMultiSigPublicKeys = GenesisBuilder.defaultPosMultiSigPublicKeys,
+          posVaultPubKey = GenesisBuilder.defaultPosVaultPubKey,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.